README.md

```diff
@@ -16,91 +16,43 @@ Once Docker is installed, run this to bootstrap the containers:
 $ docker-compose up --no-start
 ```
 
-Then run this to install the client-side and functions dependencies
-respectively:
-
-```sh
-$ docker-compose run --rm app yarn
-$ docker-compose run --rm functions yarn --cwd functions
-```
-
-Finally, you will need to add your Firebase credentials to the local
+You will need to add your Firebase credentials to the local
 project configuration. If you have not created a Firebase app to use for
 development, [do that now](https://console.firebase.google.com/).
 
 Then run this:
 
 ```sh
-$ script/firebase-setup <YOUR-FIREBASE-APP-NAME>
+$ script/firebase-setup
```

Comment:
Does this take you through some instructions in the command line? If so maybe some mention of that.